.github/workflows/helm-integ-test.yml

```diff
@@ -0,0 +1,98 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Run Helm Charts Integration Tests
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      cluster_name:
+        required: true
+        type: string
+        default: "cwagent-operator-helm-integ"
+        description: "EKS cluster name"
+      cluster_namespace:
+        required: true
+        type: string
+        default: "amazon-cloudwatch"
```

Comment:
Resources using helm will always be created in this namespace. Any specific reason to add as input instead of constant?

Reply:
Are you sure helm resources use `amazon-cloudwatch` as the default namespace? I saw helm resources being created under default when the namespace is not given with `helm_release`. Maybe there is add-on configuration to deploy helm resources to `amazon-cloudwatch` namespace?

Reply:
Makes sense.  Are we including this integ test as part of some MCM or preflight where we validate it for two namespaces default and amazon-cloudwatch? Can you help me in understanding the usecase of making it a workflow input?

Reply:
On second thought, we don't see any strong reason why we would want to use a different namespace. removed it